apps/E2E/src/Switch/specs/Switch.spec.win.ts

```diff
@@ -28,18 +27,31 @@ describe('Switch Accessibility Testing', () => {
     await SwitchPageObject.scrollToTestElement();
   });
 
-  it('Switch - Validate accessibilityRole is correct', async () => {
-    await expect(await SwitchPageObject.getAccessibilityRole()).toEqual(BUTTON_A11Y_ROLE);
+  it('Validate "accessibilityRole" defaults to Button "ControlType" element attribute.', async () => {
```

Comment:
The "Button" part is a bit confusing. What do you mean by that?

Reply:
For this test, I'm trying to say that the ControlType element attribute should be equal to a "Button" type - the value passed into accessibilityRole is "ControlType.Button"